TESTING.rst

```diff
@@ -168,19 +168,19 @@ to breeze.
 
 .. code-block:: bash
 
-     ./breeze-legacy tests tests/providers/http/hooks/test_http.py tests/core/test_core.py --db-reset -- --log-cli-level=DEBUG
+     ./breeze tests tests/providers/http/hooks/test_http.py tests/core/test_core.py --db-reset -- --log-cli-level=DEBUG
 
 You can run the whole test suite without adding the test target:
 
 .. code-block:: bash
 
-    ./breeze-legacy tests --db-reset
+    ./breeze tests --db-reset
```

Comment:
Do we have to specify `./breeze`? With autocomplete setup I think we don't need to add ./ to invoke the breeze command. @joppevos

Reply:
Thanks, you are right. I will remove it 👍